docs/shared/template-examples/taskflow.rst

```diff
@@ -17,8 +17,12 @@
 
  .. code-block:: python
 
+    from airflow.models.taskinstance import TaskInstance
+    from airflow.models.dagrun import DagRun
+
+
     @task
-    def print_ti_info(task_instance=None, dag_run=None):
+    def print_ti_info(task_instance: TaskInstance | None = None, dag_run: DagRun | None = None):
```

Comment:
I’ve had a mind to add a fix to the `=None` default (they really shouldn’t be needed), maybe I should do it now.